src/Controller/TestController.php

```diff
@@ -1,23 +1,21 @@
 <?php
```

Comment:
If we need a "Test connection" button then we should build that rather than provide something that we don't intend to maintain.  If you keep this in here - it needs to be marked as deprecated... we don't provide anything like this now - no reason to get them used to it in 8.